tests/draft2020-12/alternatives/false-vocabulary-format-assertion/is-known.json

```diff
@@ -1,9 +1,9 @@
 [
     {
         "description": "schema that uses custom metaschema with format-assertion: false",
+        "comment": "For implementations which support the format assertion vocabulary",
         "schema": {
-            "$id": "https://schema/using/format-assertion/false",
```

Comment:
Is removing the `$id` going to break things?  (cc: @karenetheridge)  (seen in several places)

Reply:
I am pretty sure no, it's just an extra unneeded validator that was in these schemas.